Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabLoader.cpp

```diff
@@ -196,7 +196,10 @@ namespace AzToolsFramework
                     }
                 }
             }
-            newTemplate.MarkAsLoadedWithErrors(isLoadedWithErrors);
+
+            isLoadSuccessful |= SanitizeLoadedTemplate(newTemplate.GetPrefabDom());
```

Comment:
This should be &= .  Otherwise, an unsuccessful load will become successful.

Reply:
Whoops. I was sure I messed up that one. Fixing!